phpBB/includes/functions_messenger.php

```diff
@@ -210,10 +210,12 @@ function set_mail_priority($priority = MAIL_NORMAL_PRIORITY)
 	/**
 	* Set email template to use
 	*/
-	function template($template_file, $template_lang = '', $template_path = '')
+	function template($template_file, $template_lang = '', $template_path = '', $template_dir_prefix = '')
 	{
 		global $config, $phpbb_root_path, $phpEx, $user, $phpbb_extension_manager;
 
+		$template_dir_prefix = (!$template_dir_prefix || substr($template_dir_prefix, 0, 1) === '/') ? $template_dir_prefix : '/' . $template_dir_prefix;
```

Comment:
`substr($template_dir_prefix, 0, 1)` is equivalent to `$template_dir_prefix[0]`, which should be faster, but in this case it is negligible in this case I guess